app/controllers/idv/socure/socure_errors_controller.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+module Idv
+  module Socure
+    class SocureErrorsController < ApplicationController
+      include Idv::AvailabilityConcern
+      include IdvStepConcern
+      include StepIndicatorConcern
+      include Idv::AbTestAnalyticsConcern
+
+      before_action :confirm_step_allowed
+      before_action :set_in_person_available
+
+      def timeout
+        @remaining_submit_attempts = rate_limiter.remaining_count
+        track_event(type: :timeout)
+      end
+
+      def go_in_person
+        attributes = {
+          remaining_submit_attempts: rate_limiter.remaining_count,
+        }.merge(ab_test_analytics_buckets)
+        analytics.idv_doc_auth_socure_choose_in_person(**attributes)
+
+        idv_session.opted_in_to_in_person_proofing = true
+        idv_session.flow_path = 'standard'
+        idv_session.skip_doc_auth_from_how_to_verify = true
```

Comment:
looking at the standard document capture controller ... should this be idv_session.skip_doc_auth_from_handoff instead?
```suggestion
        idv_session.skip_doc_auth_from_handoff = true
```

my understanding is that idv_session.skip_doc_auth_from_how_to_verify is when opting in to IPP before hybrid_handoff 🤔

Reply:
Hi!  @amirbey is correct, `skip_doc_auth_from_how_to_verify` is set to true when a user on the `verify/how_to_verify` page selects the “Verify your identity at a Post Office” option.  You can see a demo of that user path in the “After” screen recording from PR #9598.

I’m not familiar with the  logic here, so I can’t say which variable should be used on line 27 - but I did want to confirm that `idv_session.skip_doc_auth_from_how_to_verify` is set to true when a user opts in to IPP.

Reply:
i'm not clear if my suggestion is correct. it seem like we'd want to mimic the behavior of when a user opts in to IPP from the Try Again page in doc auth?

Reply:
awesome ... i tested the back button and it sent me back to /verify/socure/document_capture 🕺🏿

Reply:
it looks like if we use `idv_session.skip_doc_auth_from_handoff`, we can also set [previous_step_url](https://github.com/18F/identity-idp/blob/0dede8b0a9423a3d02496cfe172cd1c461cca1de/app/controllers/idv/document_capture_controller.rb#L110) and send the user directly where we want (ie: idv_socure_document_capture_url)

Reply:
I ended up creating a route to the document capture controller for direct access to IPP.